src/test/java/org/jitsi/testutils/ConfigPropertyTest.java

```diff
@@ -0,0 +1,123 @@
+/*
+ * Copyright @ 2018 - present 8x8, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.jitsi.testutils;
+
+import com.typesafe.config.*;
+import org.jitsi.utils.config.*;
+
+import java.util.function.*;
+
+import static org.junit.Assert.assertEquals;
+
+public class ConfigPropertyTest<T extends ConfigProperty, U>
```

Comment:
Would `T extends ConfigProperty<U>` be valid Java syntax?  Would it be helpful?  (Are there any cases where it wouldn't be valid?)

Reply:
yeah, i think that would be better and does work. fixed.